src/html-from-paste-event.js

```diff
@@ -0,0 +1,30 @@
+export function createHiddenElement () {
+  let ieHiddenElement = document.createElement('div')
+  ieHiddenElement.setAttribute('contenteditable', true)
+  ieHiddenElement.setAttribute('style', 'position: absolute; top:0; left: 0; opacity: 0;')
+  document.body.appendChild(ieHiddenElement)
+  return ieHiddenElement
+}
+
+export function removeElement (node) {
+  if (node.parentNode) {
+    node.parentNode.removeChild(node)
+  }
+}
+
+export function getHtmlUsingHiddenElement (ieHiddenElement) {
+  ieHiddenElement.focus()
+  document.execCommand('paste')
+  return ieHiddenElement.innerHTML
+}
+
+export default function htmlFromPasteEvent (event) {
+  if (window.clipboardData) { // IE11
```

Comment:
I wonder if this conditional should be the other way round. As it feels like we're doing a check of "do you support legacy feature? no, ok let's go modern" when we probably should be asking "are you modern? no, well I guess we'll see if legacy works" 

so maybe we should do 

```
const html = event.clipboardData.getData('text/html')

if (!html && window.clipboardData)
```

What do you think?

I don't know if there were any browsers that got caught in the middle of window.clipboardData and event.clipboardData though so this could be rather moot.

Reply:
Swapped the conditional statement